### ISSUE TITLE
The training set of the model

### ISSUE THREAD

**Issue by wjczf123** [zhengxxn/adaptive-knn-mt#11]
Hi, xin.
I have a question about the training set of the model.
Do you directly use the dev set of IT domain to train model and use test set of IT domain to test.
This means we only need few in-domain samples to train the meta-network.

**Comment by zhengxxn** [comment]
yes

**Comment by wjczf123** [comment]
Thanks for your reply. Best regards!
